README.md

```diff
@@ -52,6 +52,22 @@ You can run the application with any ASGI server, including [uvicorn](http://www
 
 <p align="center">&mdash; ⭐️ &mdash;</p>
 
+* [Responses](#responses)
+    * [Response](#response)
+    * [HTMLResponse](#htmlresponse)
+    * [PlainTextResponse](#plaintextresponse)
+    * [JSONResponse](#jsonresponse)
+    * [StreamingResponse](#streamingresponse)
+    * [FileResponse](#fileresponse)
+* [Requests](#requests)
+    * [Request](#request)
+* [WebSocket](#websocket)
+* [Routing](#routing)
+* [Static Files](#static-files)
+* [Test Client](#test-client)
+* [Debugging](#debugging)
+* [Decorators](#decorators)
```

Comment:
Yup this'd be great - let's definately get a table of contents in.
PR much appreciated!